test/e2e/fixture/cluster/context.go

```diff
@@ -12,11 +12,12 @@ import (
 type Context struct {
 	t *testing.T
 	// seconds
-	timeout int
-	name    string
-	project string
-	server  string
-	upsert  bool
+	timeout    int
+	name       string
+	project    string
+	server     string
+	upsert     bool
+	namespaces []string
```

Comment:
Since namespaces is the only attribute added, why all the other ones are showing a diff ?

Reply:
VS code indentation.